runtime/jcl/common/system.c

```diff
@@ -356,11 +358,23 @@ jobject getPropertyList(JNIEnv *env)
 		}
 		strings[propIndex++] = !usernameAlloc || result ? "unknown" : usernameAlloc;
 	}
-
 #undef USERNAME_LENGTH
 
+	/* Set the maximum direct byte buffer allocation amount */
+
+	UDATA heapSize = javaVM->memoryManagerFunctions->j9gc_get_maximum_heap_size(javaVM);
```

Comment:
This won't compile on various platforms (.c file, not .cpp).

Reply:
Sad but true.  Fixed.